### ISSUE TITLE
Define token taxonomy and initial tokens

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#1973]
## Description
We need to define the taxonomy for our tokens and the initial set of tokens we'll have in the VA Design System. We currently have foundational tokens but no semantic meaning to those tokens nor a clear taxonomy for building them.

## Details
I'm going to follow the recommendations in this article: https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676#312d

At the moment I'm leaning towards:

System - Domain - Category - Concept - Property - Variant

For example: vads-core-font-family-sans

Some rules (from the article) that we'll follow:
* No single token includes all potential levels. Some levels — domain, theme, element — are rarely needed. Other levels — a quantitative scale versus qualitative variant — tend to be mutually exclusive. 
* Avoid dogmatically including all levels possible or duplicating token-tuples redundantly. Instead, include only the levels needed to sufficiently describe and distinguish purposeful intent.

Thus we can also have a token like this: vads-color-text-primary

## Tasks
- [x] Define token taxonomy.
- [x] Define design tokens.
- [x] Hand off definition to development team.

## Acceptance Criteria
- [x] DST designers and developers have a document that details the tokens and their structure

**Comment by humancompanion-usds** [comment]
I've started to define primitive, and a few semantic, [design tokens](https://airtable.com/invite/l?inviteId=invrjoqpGSqjdYf4s&inviteToken=1082a3b62e3d39fe955ce43eaa5d6c484789e7e7f96c8df1cd7f548efab060fd&utm_medium=email&utm_source=product_team&utm_content=transactional-alerts) Making slow but steady progress. I'm focused on spacing first and then I'll move onto color and typography. I have the taxonomy defined.

**Comment by humancompanion-usds** [comment]
I have the spacing values defined that are found in our tokens file, minus the negative values, which are all just a copy of the USWDS values.

I also have all of the colors defined that are found in our tokens file. However, [there are more colors found in Formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-variables.scss) and I'm unclear whether all of those are in use or not. A good portion of those are accounted for, and [some additional color mappings are in the Sketch file from USWDS](https://sketch.com/s/95194b76-8153-4844-8be1-2e9a25d6d321). Notably the following:

* Aqua
* Red
* Cool blue
* Purple

That file also features the Hub colors which I'm surprised to not find in our current tokens file. So I guess my question is: Is our current tokens file up-to-date? Is it missing a fair number of colors that are defined AND in use in Formation? If so we may need to expand our tokens to include those colors.

My next step is to add semantic token color values that I'd like to include in v3 and beyond. That won't be done until I return from vacation on 9/5/23.

**Comment by humancompanion-usds** [comment]
I added some semantic tokens and started on typography. If we can run a script, which I think Andrew has done, then more colors could be added. The other piece of work here is that I'd like both a current-MVP tokens file and a v3 tokens file so we can load the latter into Figma. I'm not sure which values we've used in what Alex has done in getting tokens into Figma but it should be the v3 values.
